src/httpClient.ts

```diff
@@ -101,21 +101,72 @@ export default class httpClient {
         };
     }
 
-    async postMetaAnalysis(records: AirtableRecord[], aggregation_variable: AggregationFactor, meta_analysis_technique: string = 'fixed', meta_analysis_transformation: string = 'double_arcsin_precise'){
+    async getEstimateGrades(records: AirtableRecord[]) {
+        const formattedRecords = records.map(o => {
+            return {
+                country: o.country,
+                estimate_grade: o.estimate_grade,
+                serum_pos_prevalence: o.seroprevalence,
+                denominator: o.denominator
+            }
+        })
+        const req_body = {
+            records: formattedRecords
+        }
+        const response = await this.httpPost('/airtable_scraper/country_seroprev_summary', req_body);
+        if (!response) {
+            return [];
+        }
+        const airtable_records = response.map((item: Record<string, any>) => {
```

Comment:
not "airtable_records" anymore persay, can we call this estimate_grades or estimate_metadata?